Dockerfile

```diff
@@ -14,22 +15,28 @@ ARG DEBIAN_FRONTEND=noninteractive
 RUN apt-key del 7fa2af80
 RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/3bf863cc.pub
 RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu2004/x86_64/7fa2af80.pub
+# Update the base image
+RUN apt update && apt upgrade -y
+# Install bittensor
+## Install dependencies
+RUN apt install -y curl sudo nano git htop netcat wget unzip python3-dev python3-pip tmux apt-utils cmake build-essential
+## Upgrade pip
+RUN pip3 install --upgrade pip
 
-RUN apt-get update && apt-get install --no-install-recommends --no-install-suggests -y apt-utils curl git cmake build-essential unzip python3-pip  wget iproute2 software-properties-common
+# Install nvm and pm2
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash
```

Comment:
Can you check here the expected hash of the content you are downloading?
An alternative could be pinning here the file you need

Reply:
I think we can just do pip install now, no?

Reply:
yes yes, we can generate the docker image after generating the wheel